node/tests/RedisClient.test.ts

```diff
@@ -72,6 +60,25 @@ describe("RedisClient", () => {
         };
     };
 
+    runBaseTests<Context>({
+        init: async (protocol, clientName?) => {
+            const options = getOptions(cluster.ports()[0], protocol);
```

Comment:
minor: save `cluster.ports()[0]` in the describe scope as `port`, like it was done before.